dist/tools/isrvecgen/isrvecgen.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Tool for automatically generating interrupt vector tables from CMSIS vendor headers
+#
+# Author:   Hauke Petersen <hauke.petersen@fu-berlin.de>
+
+import argparse
+import sys
+import os
+import glob
+import re
+import copy
+from operator import itemgetter
+
+cpulist = set()
```

Comment:
Same comment for this global variable. By the way, the same variable name is used internally in other functions, this looks weird:
```python
for cpulist in specific:
    ...
```
for example